src/view/components/formikReactSpectrum3/radioGroupWithDataElement.jsx

```diff
@@ -38,7 +38,6 @@ const RadioGroupWithDataElement = ({
   name,
   children,
   dataElementDescription,
-  width,
```

Comment:
Having `width` on this component didn't really do what was expected because we were forwarding on the width to the ReactSpectrumRadioGroup and the data element textfield, which is indented. Meaning the total width of the component ends up being larger than defined width. Everywhere we're currently using RadioGroupWithDataElement seems to look better without a constrained width as well. Because of these things, I just decided to remove the prop entirely and if we want to add support for width later, we'll need to be smarter about how we accomplish it.